blocks/init/src/Blocks/components/image/manifest.json

```diff
@@ -108,11 +108,11 @@
 			},
 			"imageAspectRatio": {
 				"twClasses": {
-					"square": "aspect-square",
-					"three-two": "aspect-3/2",
-					"five-four": "aspect-5/4",
-					"sixteen-nine": "aspect-video",
-					"twentyone-nine": "aspect-21/9"
+					"square": "aspect-square w-full",
```

Comment:
This should be set in the block, not the component

Components should be neutral sizing-wise, if possible